tests/test_doctestplus.py

```diff
@@ -775,3 +775,156 @@ def test_doctest_skip(testdir):
         """
     )
     testdir.inline_run(p, '--doctest-plus', '--doctest-rst').assertoutcome(skipped=1)
+
+
+def test_remote_data_on(testdir):
+    testdir.makeini(
+        """
+        [pytest]
+        doctestplus = enabled
+    """)
+
+    p = testdir.makefile(
+        '.rst',
+        """
+        # This test should pass
+        .. doctest-remote-data::
+
+            >>> 1 + 1
+            2
+        """
+    )
+    testdir.inline_run(p, '--doctest-plus', '--doctest-rst',
+                       '--remote-data').assertoutcome(passed=1)
+
+
+def test_remote_data_off(testdir):
+    testdir.makeini(
+        """
+        [pytest]
+        doctestplus = enabled
+    """)
+
+    p = testdir.makefile(
+        '.rst',
+        """
+        # This test should fail, but we skip it, thus the test should be skipped.
```

Comment:
The test would pass with `--remote-data` I think ? I don't see what would make it fail.

Reply:
My bad, I was making  a lot of little changes here and there in this PR, that I forgot to change the test comments after adjusting the tests. Previously, we had some test lines that should fail to help us ascertain that the test was indeed skipped, so that if the test was not skipped then it should fail. But I removed them when I realized that the assertion pretty much does that already.

Reply:
Sure, no worries, we do reviews also to catch those little inconsistencies.